solidity/deploy/05_deploy_bank.ts

```diff
@@ -7,8 +7,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { deployer } = await getNamedAccounts()
 
   const bank = await deploy("Bank", {
-    contract:
-      process.env.TEST_USE_STUBS_TBTC === "true" ? "BankStub" : undefined,
+    contract: process.env.TEST_USE_STUBS_TBTC === "true" ? "BankStub" : "Bank",
```

Comment:
Shouldn't we do a similar change for `deployProxy` function in `06_deploy_bridge.ts`?
```
contractName:
        process.env.TEST_USE_STUBS_TBTC === "true" ? "BridgeStub" : "Bridge",
```

Reply:
Makes sense, please see fc271946885d8ba4cfbff5160e52ab99c6706da0.